test/integration/serverTest.js

```diff
@@ -225,6 +259,30 @@ describe('Brave Sensor server', () => {
         auth: helpers.getEnvVar('PARTICLE_ACCESS_TOKEN'),
       })
     })
+
+    it('should send a 200 response if a sirenAddressed call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await sirenAddressedAlert('particleCoreIdTest')
+      expect(response).to.have.status(200)
+    })
+
+    it('should send a 200 response if a sirenEscalated call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await sirenEscalatedAlert('particleCoreIdTest')
+      expect(response).to.have.status(200)
+    })
+
+    it('should make the chatbot state completed after an addressed alert', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      await sirenAddressedAlert('particleCoreIdTest')
+      const session = await db.getMostRecentSession(testLocation1Id)
+      expect(session.chatbotState).to.equal(CHATBOT_STATE.COMPLETED)
+    })
+
+    it('should not call startAlertSession if particleSirenID is not null', async () => {
+      const sessions = await db.getAllSessionsFromLocation(testLocation1Id)
+      expect(sessions.length).to.equal(0)
+    })
```

Comment:
We should also have tests to answer the questions:
- What happens with the given Siren Core ID isn't in the DB?
- What happens if not given a Siren Core ID
- Is the UpdatedAt value updated?
- Is the SendSingleAlert function called with the right parameters?